### DISCUSSION TITLE
CUSTOM ARROWS

### DISCUSSION THREAD

**Discussion by fayzali1** [akiran/react-slick#2177] [Q&A]
Hey everyone, can somebody guide me with using custom arrows with react-slick, I passed the customs to the props the are working fine but they are aligned vertically with the track, is there some kind of prop I'll have or some css override? Very sorry if I was not supposed to ask this here .

**Comment by geeky01adarsh** [comment]
Find the solution here:- https://github.com/akiran/react-slick/blob/master/examples/CustomArrows.js
